packages/manager/modules/nutanix/src/node.class.js

```diff
@@ -0,0 +1,17 @@
+import { NODE_STATUS } from './constants';
+
+export default class Cluster {
```

Comment:
```suggestion
export default class Node {
```

Reply:
Seen together. Cluster already exists but is in conflict with API data